src/epr/config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: © 2024 Brett Smith <xbcsmith@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List
+
+
+@dataclass
+class Config:
+    """Data class for Config"""
+
+    url: str
+    token: str
+    debug: bool = False
+
+    events: List[Dict[str, Any]] = field(default_factory=list)
+    event_receivers: List[Dict[str, Any]] = field(default_factory=list)
+    event_receiver_groups: List[Dict[str, Any]] = field(default_factory=list)
```

Comment:
Should be lists of respective types from `models` to match usage. `main.py` has insertions like
```python
event = Event()
...
cfg.events.append(event)
```

Reply:
Decided the library should not require the overhead of the Dataclasses.

Reply:
If that's the case then please call `.as_dict()` on `main.py`'s lines 518, 528, & 538 (where these lists are appended to).

Reply:
decided you were right